resources/todomvc/readme.md

```diff
@@ -1,142 +1,30 @@
-Pascal is a Software Engineer at weluse GmbH with a deep passion for consistency. He watches pull requests and helps developers getting their contributions integrated with TodoMVC.
-
-#### [Stephen Sawchuk](https://github.com/stephenplusplus) - Developer
-
-<img align="left" width="40" height="40" src="https://secure.gravatar.com/avatar/098cfe2d360e77c3229f2cd5298354c4?s=40">
-Stephen is a Front-end Engineer at Quicken Loans that cares about improving the maintainability and developer experience of open-source projects. His recent contributions include helping us move all apps over to using Bower and implementing the new information bar.
-
-#### [Gianni Chiappetta](https://github.com/gf3) - Logo designer
-
-<img align="left" width="40" height="40" src="http://www.gravatar.com/avatar/4b0209ae3652cc5a7d53545e759fbe39.png?s=40">
-Gianni is a programmer and designer currently working as the Chief Rigger at MetaLab.
-
-## Disclaimer
-
-<img align="right" width="230" height="230" src="https://raw.github.com/tastejs/todomvc/gh-pages/media/icon-small.png">
-
-TodoMVC has been called many things including the 'Speed-dating' and 'Rosetta Stone' of MV* frameworks. Whilst we hope that this project is able to offer assistance in deciding what frameworks are worth spending more time looking at, remember that the Todo application offers a limited view of what a framework may be capable of.
-
-It is meant to be used as a gateway to reviewing how a basic application using a framework may be structured and we heavily recommend investing time researching a solution in more depth before opting to use it.
-
-
-## Getting Involved
+![screenshot](./screenshot.png)
 
-Whilst we enjoy implementing and improving existing Todo apps, we're always interested in speaking to framework authors (and users) wishing to share Todo app implementations in their framework/solution of choice.
+## What are we testing
 
-Check out our [contribution docs](contributing.md) for more info.
+- Stress-test DOM manipulations of a repeated action.
+- Impact of JavaScript version releases.
+- Tools (bundlers & transpilers) for build optimizations.
+- Libraries & frameworks for render strategies and architectural patterns
 
+## How are we testing
 
-## License
+The test that repeats a set number of times allows us to stress-test DOM manipulation by adding, updating and removing an element. This simulates a real-world user flow that is common for a web application. While the test is running, the benchmark tool captures performance metrics, to be able to compare the different workloads (apps).
 
-Everything in this repo is MIT License unless otherwise specified.
+The test consist of the following steps:
 
-MIT © Addy Osmani, Sindre Sorhus, Pascal Hartig, Stephen Sawchuk.
+- Wait for the “add task” input field to be present in the DOM
+- Add a task
+- Check off a task
+- Delete a task
+- Repeat steps 2 - 4 a set number of times (100 by default).
```

Comment:
Nit: This is a little ambiguous between whether it's repeating the steps internally (doing step 2 100 times, ...) or across steps (doing step 2-4 100 times). My understanding is it's the former, and so I might rephrase this as something like:

```
- Wait for the “add task” input field to be present in the DOM
- Add 100 tasks
- Check off all tasks
- Delete all tasks
```

Reply:
I'm referring to this:
https://github.com/WebKit/Speedometer/blob/main/resources/tests.mjs#L29

And If I'm reading it correctly it does:
1. add a task (100x)
2. check off a task (100x)
3. delete a task (100x)

Reply:
@bgrins - your proposed change made it a lot clearer 😄